src/global/less/corporate-ui-light.less

```diff
@@ -0,0 +1,15 @@
+
+/*!
+ * Corporate UI
+ * Maintained and developed by IXCC : corporate-ui.slack.com
+ */
+
+// Insert Compile Date
+@compiled_at: `new Date()`;
+.compiled_at {content: "@{compiled_at}";}
+
```

Comment:
Probably add fontawesome in corporate-ui-light.less or somewhere else to support "fal fa-" in cui-light
@import url("https://static.scania.com/resources/icons/font-awesome/5.3.1/css/all.min.css");